### DISCUSSION TITLE
Campo TGUID

### DISCUSSION THREAD

**Discussion by mnmurari** [viniciussanchez/dataset-serialize#199] [Q&A]
Tenho uma tabela onde o PK é um TGUID como esse {54699ED2-6B8C-462C-BA82-EBD5856665F5}.
Quando uso o comando qryAux.ToJSONArray(), ele não retorna esse campo.
Fiz um teste e no nome do cliente coloquei {marcelo}, ele também não retornou.

Como fazer para que esse comando .ToJSONArray() retorne os campos mesmo que estejam entre {}

Abraços e obrigado
Marcelo

**Comment by viniciussanchez** [comment]
Como está ficando o JSON gerado? Ele simplesmente ignora o campo?
